### PR TITLE
Add online help for Alignment and Normalization modules

### PR DESCRIPTION
**Pull request by omokshyna**
Removed offline and added online help for Alignment and Normalization modules

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_adap3/ADAP3AlignerParameters.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public class ADAP3AlignerParameters extends SimpleParameterSet {
   public static final DoubleParameter SAMPLE_COUNT_RATIO = new DoubleParameter(
       "Min confidence (between 0 and 1)",
       "A fraction of the total number of samples. An aligned feature must be detected at "
-          + "least in several samples. This parameter determines the minimum number of samples where a "
+          + "least in several samples.\nThis parameter determines the minimum number of samples where a "
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/modules/dataprocessing/align_adap3/ADAP3AlignerParameters.java:48]: more of a general comment - no need to apply it everywhere, but you can look at text blocks which is kind of nice for longer text

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_hierarchical/HierarAlignerGCParameters.java [hunk 2]
```diff
@@ -42,44 +42,44 @@ public class HierarAlignerGCParameters extends SimpleParameterSet {
...
-  public static final BooleanParameter saveRAMratherThanCPU_1 = new BooleanParameter(
-      "Save RAM (!! <FULL-EXPENSE-ON-CPU> !!)",
-      "Ignored if clusterer type is not 'Cached'. Saves RAM at the expense of CPU during the whole clustering process (NO precomputed distances).",
-      false);
-  // -- Save RAM 2
-  public static final BooleanParameter saveRAMratherThanCPU_2 = new BooleanParameter(
-      "Save RAM (!! <LAST-PART-ONLY> !!)",
-      "Ignored if clusterer type is not 'Cached'. Saves RAM at the expense of CPU during the 2nd step of the clustering (building clusters from tree).",
-      false);
+//  public static final BooleanParameter saveRAMratherThanCPU_1 = new BooleanParameter(
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/modules/dataprocessing/align_hierarchical/HierarAlignerGCParameters.java:45]: you can remove all commented out code directly

**Review by robinschmid — APPROVED** [review]

**Review by robinschmid — APPROVED** [review]
I think its good like it is.

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_hierarchical/HierarAlignerGCParameters.java (hunk 2 above)
Comment (omokshyna) [reply src/main/java/io/github/mzmine/modules/dataprocessing/align_hierarchical/HierarAlignerGCParameters.java]: Done

**Pull request merged**